api/illumio/cloud/k8scluster/v1/resource_config.proto

```diff
@@ -94,8 +94,25 @@ message KubernetesResourceMutation {
   }
 }
 
+enum LogLevels {
+  LOG_LEVELS_INFO_UNSPECIFIED = 0;
+  LOG_LEVELS_DEBUG = 1;
+  LOG_LEVELS_ERROR = 2;
+  LOG_LEVELS_PANIC = 3;
+}
+
+message KubernetesLogsRequest {
+  string logs = 1;
+}
```

Comment:
1) Keep the little structure and metadata we still get in a Zap hook, cf. [the `Entry` type](https://github.com/uber-go/zap/blob/d6b2d14fc541ed17b31b794afc139f03648842ca/zapcore/entry.go#L143-L150).
2) Remove the `Kubernetes` prefix as this doesn't have anything to do with k8s.
3) Future-proof the API by making the request a `oneof`, like `SendKubernetesResourcesRequest`, so we can send new types of messages into this stream later.
```suggestion
message Log {
  LogLevel level = 1;
  google.protobuf.Timestamp time = 2;
  string message = 3;
}

message SendLogsRequest {
  oneof request {
    Log log = 1;
  }
}
```

Unfortunately, in an `Entry`, the message is already serialized, i.e. we have already lost the structure (message + fields). But if you use a JSON encoder, the `message` can still be a JSON object, so it will be somewhat structured.

Reply:
I'd actually name the `message` field `json_message` to clarify that this contains a JSON object.